wiki/markdown/running-eiffel-intelligence-frontend.md

```diff
@@ -0,0 +1,44 @@
+# Running Eiffel Intelligence Front-end
+
+## _Prerequisites_
+
+Eiffel intelligence Front-end is a Spring microservice distributed in a war file. To start
+the service download the war file from
+[jitpack.io](https://jitpack.io/#eiffel-community/eiffel-intelligence-frontend) and look for the
+latest version. Now replace the latest version in the link below:
+
+    https://jitpack.io/com/github/eiffel-community/eiffel-intelligence-frontend/<version>/eiffel-intelligence-<version>.war
+
+If you want to test the latest code in github clone the project and compile it
+with:
+
+    mvn clean install
+
+append **_-DskipTests_** if you want to skip the tests since the latest on
+master always has passed the tests. The war file should now be found under
+target folder in your cloned project.
+
+If you run from source code, you can run Eiffel-Intelligence Front-end with maven command:
+
+    mvn spring-boot:run
+
+Or you can run executable war file (located in target folder, if running from source code):
+
+    java -jar eiffel-intelligence-frontend-<version>.war
+
+if you want to run with default configuration for local message buss and
```

Comment:
buss --> bus

Reply:
fixed,, removed that due to it is not valid for frontend. This file is based on same file from EI Backend, with some modifications for frontend usage point-of-view.